src/entityManager.ts

```diff
@@ -420,6 +420,104 @@ export function initialize(): void {
     new Sensor(EntityNames.PM_IN_HUMIDITY_AQIN, deviceId, SensorUnit.percent, DeviceClass.HUMIDITY),
   );
 
+  // Calculated sensors
+  entities.set(
+    EntityNames.SOLARRADIATION_LUX,
+    new Sensor(EntityNames.SOLARRADIATION_LUX, deviceId, SensorUnit.illuminance, DeviceClass.ILLUMINANCE),
```

Comment:
Not sure if intentional, but missing SensorUnit. Presumably would have to define a new one

Reply:
Did you comment on the wrong line? This has a sensor unit of illuminance

Reply:
Just overlooked it. My bad. Looks good